index.html

```diff
@@ -95,12 +95,12 @@ <h4>Wins</h4>
                     <div class="card_main_program">
                         <i class="fa-solid fa-user-plus"></i>
                         <h4>Deliver</h4>
-                        <p>With all the map of challenges, opportunities and process we activate execution!</p>
+                        <p>With the map of challenges, opportunities and process we activate execution!</p>
```

Comment:
- [ ] Great job with the cards. Please kindly implement the hover effect on the cards, which brings out the border.
- [ ] please also ensure your cards can be seen, as all a user sees for now is just the text and icon. They should be in separate cards. You can give a slightly different background color.
- [ ] you are missing the `see the whole program` underline link below the cards. Please add that

yours
![Screenshot (620)_LI](https://user-images.githubusercontent.com/65298048/203653860-f60931eb-e36d-4fdc-8db3-c9117384cb5d.jpg)
Behance design
![Screenshot (619)](https://user-images.githubusercontent.com/65298048/203653951-33b6cf0d-31f0-439b-b57c-8d4aee5ab4a7.png)